### ISSUE TITLE
cannot install nuget package

### ISSUE THREAD

**Issue by jordiserres0** [ConvertAPI/convertapi-dotnet#10]
VS2017: On opening the Nugget Manager Console and typing
> Install-Package ConvertApi
I receive the following message in red background:
Restoring packages for D:\Tools\WordToPdf\convertapi-dotnet-master\ConvertApi\ConvertApi.csproj...
Install-Package : NU1108: Cycle detected. 
  ConvertApi -> ConvertApi (>= 2.3.0).
At line:1 char:1
+ Install-Package ConvertApi
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Install-Package : Package restore failed. Rolling back package changes for 'ConvertApi'.
At line:1 char:1
+ Install-Package ConvertApi
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand

**Comment by tomasr78** [comment]
Are you trying to update package previously installed?
What type of .NET project is it? .NET version?

**Comment by jordiserres0** [comment]
No. I'm just trying to execute a conversion from Word to PDF for the first time

**Comment by jordiserres0** [comment]
.net version is now 4.6

Trying to install it in a project using VS2015 the result is:

 

PM> Install-Package ConvertApi

Attempting to gather dependency information for package 'ConvertApi.2.3.0' with respect to project 'FBOExplorer', targeting '.NETFramework,Version=v4.6'

Gathering dependency information took 1.08 sec

Attempting to resolve dependencies for package 'ConvertApi.2.3.0' with DependencyBehavior 'Lowest'

Resolving dependency information took 0 ms

Resolving actions to install package 'ConvertApi.2.3.0'

Resolved actions to install package 'ConvertApi.2.3.0'

Retrieving package 'ConvertApi 2.3.0' from 'nuget.org'.

Install failed. Rolling back...

Package 'ConvertApi.2.3.0' does not exist in project 'FBOExplorer'

Package 'ConvertApi.2.3.0' does not exist in folder 'D:\Projects2015\packages'

Executing nuget actions took 189.57 ms

Install-Package : Could not install package 'ConvertApi 2.3.0'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.6', but 

the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

At line:1 char:1

+ Install-Package ConvertApi

+ ~~~~~~~~~~~~~~~~~~~~~~~~~~

    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception

    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
